dangvuthang/COSC2769-rmit
# Feat/ update room

Fully functional update function.

**How to test**
- Checkout PR
- Login with the expert account (abcxyz@gmail.com \ 10102001)
- Navigate to /dashboard
- Click on the edit room button
- Fill out the edit form and press the Update button
- Verify the information

**Issue**
1. We do not have a fixed image aspect ratio for the room so uploading photos will not be easy.
2. I have not implemented an image manipulating library.

close #73
